src/cascade/dismod/serialize.py

```diff
@@ -2,25 +2,103 @@
+
+    """
+    bundle_fit = DismodFile()
+
+    # Standard Density table.
+    density_enum = enum_to_dataframe(DensityEnum)
+    densities = pd.DataFrame({"density_name": density_enum["name"]})
+    bundle_fit.density = densities
+
+    # Standard integrand naming scheme.
+    all_integrands = default_integrand_names()
+    bundle_fit.integrand = all_integrands
+
+    bundle_fit.covariate = bundle_fit.empty_table("covariate")
+    LOGGER.debug(f"Covariate types {bundle_fit.covariate.dtypes}")
+
+    # Defaults, empty, b/c Brad makes them empty.
+    bundle_fit.nslist = bundle_fit.empty_table("nslist")
+    bundle_fit.nslist_pair = bundle_fit.empty_table("nslist_pair")
+    bundle_fit.mulcov = bundle_fit.empty_table("mulcov")
+
+    bundle_fit.log = make_log_table()
+
+    bundle_fit.node = make_node_table(model)
+
+    non_zero_rates = [rate.name for rate in model.rates if rate.parent_smooth or rate.child_smoothings]
+    if "iota" in non_zero_rates:
+        if "rho" in non_zero_rates:
+            value = "iota_pos_rho_pos"
+        else:
+            value = "iota_pos_rho_zero"
+    else:
+        if "rho" in non_zero_rates:
+            value = "iota_zero_rho_pos"
+        else:
+            value = "iota_zero_rho_zero"
+    bundle_fit.option = pd.DataFrame(
+        {
+            "option_name": ["parent_node_name", "rate_case"],
+            "option_value": [bundle_fit.node.iloc[0]["node_name"], value],
+        }
+    )
+
+    bundle_fit.data = make_data_table(model)
+
+    # Include all data in the data_subset.
+    bundle_fit.data_subset = pd.DataFrame({"data_id": np.arange(len(bundle_fit.data))})
+
+    # Ages and times are used by Weight grids and smooth grids,
+    # so pull all ages and times from those two objects in the
+    # internal model. Skip weight grid here b/c assuming use constant.
+    bundle_fit.age = make_age_table(model)
+    bundle_fit.time = make_time_table(model)
+
+    bundle_fit.weight, bundle_fit.weight_grid = simplest_weight()
+
+    bundle_fit.prior, prior_id_func = make_prior_table(model, bundle_fit.density)
+    bundle_fit.smooth, bundle_fit.smooth_grid, smooth_id_func = make_smooth_and_smooth_grid_tables(
+        model, bundle_fit.age, bundle_fit.time, prior_id_func
+    )
+
+    def integrand_id_func(name):
+        return int(bundle_fit.integrand.query("integrand_name==@name").integrand_id)
+
+    # The avgint needs to be translated.
+    bundle_fit.avgint = make_avgint_table(model, integrand_id_func)
+
+    bundle_fit.rate = make_rate_table(model, smooth_id_func)
+
+    return bundle_fit
+
+
 def enum_to_dataframe(enum_name):
     """Given an enum, return a dataframe with two columns, name and value."""
     return pd.DataFrame.from_records(
         np.array(
             [(measure, enum_value.value) for (measure, enum_value) in enum_name.__members__.items()],
-            dtype=np.dtype([('name', object), ('value', np.int)])
+            dtype=np.dtype([("name", object), ("value", np.int)]),
```

Comment:
I have noticed that you're saying that lists should end with a comma

Reply:
Yes. I put them most places, the pretty printer puts them _everywhere_. Trailing commas make it very fast to add stuff to the list or move items around. It also makes diffs shorter for simple changes in the list.